src/runner/browser-job.js

```diff
@@ -1,13 +1,13 @@
 import Promise from 'pinkie';
-import { EventEmitter } from 'events';
 import { remove } from 'lodash';
+import Emittery from '../utils/async-emitter';
 import TestRunController from './test-run-controller';
 import SessionController from '../test-run/session-controller';
 import RESULT from './browser-job-result';
 
 
 // Browser job
-export default class BrowserJob extends EventEmitter {
+export default class BrowserJob extends Emittery {
```

Comment:
Just curious, what is the problem with synchronous EventEmitter, we have some specific case or it's just a optimisation?